cineast-core/src/main/java/org/vitrivr/cineast/core/features/OCRSearch.java

```diff
@@ -25,3 +79,221 @@ protected String enrichQueryTerm(String queryTerm) {
     return queryTerm + "~1";
   }
 }
+
+class TesseractRecognizer {
+
+  public String recognizer(BufferedImage image){
+    Tesseract tesseract = new Tesseract();
+    tesseract.setDatapath("resources/OCRSearch/TesseractData/tessdata/");
+    tesseract.setLanguage("eng");
+    tesseract.setTessVariable("tessedit_char_whitelist", "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890");
+    tesseract.setPageSegMode(6);
+    tesseract.setOcrEngineMode(2);
+    tesseract.setTessVariable("user_defined_dpi", "300");
+    try {
+      String result = tesseract.doOCR(image);
+      return result;
+    } catch (TesseractException e) {
+      e.printStackTrace();
+      return null;
+    }
+  }
+}
+
+class AttentionOCRrecognizer{
+  private static final String modelpath = "resources/OCRSearch/AttentionOCRGraph/aocr_frozen_graph/";
+  protected static byte[] graphDef;
+  protected static byte[] imageBytes;
+
+  private static byte[] readAllBytesOrExit(Path path)  {
+    try {
+      return Files.readAllBytes(path);
+    } catch (IOException e) {
+      System.err.println("Failed to read [" + path + "]: " + e.getMessage());
+      System.exit(1);
```

Comment:
Please introduce transparent error handlin (i.e. throwing the exception upwards in the stack) rather than using System.exit() - other features might work nevertheless